packages/elements/src/components/ino-avatar/ino-avatar.tsx

```diff
@@ -53,6 +61,27 @@ export class Avatar implements ComponentInterface {
    * https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/img_role.
    */
   @Prop() a11yLabel?: string = '';
+  /**
+   * Overrides the avatar's loading animation behavior.
+   * When set to true, the loading animation is displayed indefinitely.
+   * When set to false, the avatar will not show any loading animations.
+   * 
+   * By not using this property or setting it to undefined, the loading animation will be shown only while the image is being loaded.
+   */
+  @Prop() overrideLoading?: boolean = false;
```

Comment:
I would name it `isLoading` and rename our internally used variable (e.g. `imgIsFetching` or `_isLoading`)